Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -65,8 +65,8 @@ def check_xcode_up_to_date
         return unless MacOS::Xcode.installed? && MacOS::Xcode.outdated?
 
         message = <<-EOS.undent
-          Your Xcode (#{MacOS::Xcode.version}) is outdated
-          Please update to Xcode #{MacOS::Xcode.latest_version}.
+          Your Xcode (#{MacOS::Xcode.version}) is outdated.
+          Please update to Xcode #{MacOS::Xcode.latest_version} (or delete it).
```

Comment:
That's going to lead to people deleting it when they shouldn't, and to people opening issues to ask whether they should delete it.

This could be offset somewhat if the docs had a page that described the differences between Xcode and the CLT's, and describes clearly under what circumstances one should install which one(s).  This is sorely needed anyway, I think.  I myself don't know whether I should have the CLT if I already have Xcode, otherwise I'd make the page and open a PR.